src/gui/controls/volumecontrol.cpp

```diff
@@ -206,6 +220,54 @@ void VolumeControlPrivate::updateDisplay(double volume) const
     }
 }
 
+void VolumeControlPrivate::updateToolTip(int value)
+{
+    if(!m_toolTip) {
+        m_toolTip = new ToolTip(m_self->window());
+        m_toolTip->show();
+    }
+
+    constexpr auto MinVolumeLog10 = -2;
+    if(value > (MinVolumeLog10 * m_volumeSlider->scale())) {
+        m_toolTip->setText(QString(u8"%1 dB").arg(20 * (value / m_volumeSlider->scale()), 0, 'f', 1));
+    }
+    else {
+        m_toolTip->setText(QString(u8"-∞ dB"));
+    }
```

Comment:
Using `QStringLiteral` for setting the tooltip text is preferred here, as it will save on allocation at runtime.